src/Core/src/Handlers/SwipeView/SwipeViewHandler.Windows.cs

```diff
@@ -1,15 +1,17 @@
 using System;
-using System.Linq;
-using Microsoft.Maui.Graphics;
-using WSwipeControl = Microsoft.UI.Xaml.Controls.SwipeControl;
-using WSwipeItem = Microsoft.UI.Xaml.Controls.SwipeItem;
-using WSwipeItems = Microsoft.UI.Xaml.Controls.SwipeItems;
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class SwipeViewHandler : ViewHandler<ISwipeView, WSwipeControl>
+	public partial class SwipeViewHandler : ViewHandler<ISwipeView, MauiSwipeView>
```

Comment:
We can't change the type on the `SwipeViewHandler`
We'll need to figure out the strategy here for changing these types. 

Most likely we'll need to create a new handler and obsolete this one.

Reply:
We can't in .NET 7 but we can do it for .NET 8, right?

Reply:
Would it be possible to offer a variation via CommunityToolkit.Maui? Waiting another year for a bug fix is a problem...